### ISSUE TITLE
Create a Icon/Logo to Chapter App

### ISSUE THREAD

**Issue by rosanacoronado** [bitDubai/fermat-graphic-design#1053]

**Comment by rosanacoronado** [comment]
Para todas las propuesta se trabaja la idea de representar capitulos, los cual los asocie con elementos de hoja de papel, y y los superpuse de manera que represente una serie de capitulos. Se utilizo la paleta del logo de fermat, a su vez hago uso del elemento de la pica que muy comunmente representa el proceso de minado. 

# Propuesta_1 Variedad de versiones
![logo_chapter_app_v1-01](https://cloud.githubusercontent.com/assets/19508456/22356353/076e391e-e407-11e6-9e13-bae2983f42ad.png)


# Propuesta_2
![logo_chapter_app_v2-01](https://cloud.githubusercontent.com/assets/19508456/22356420/9b04c8dc-e407-11e6-9e1b-782cbef75463.png)


# Propuesta_3
![logo_chapter_app_v3-01](https://cloud.githubusercontent.com/assets/19508456/22356350/072f024e-e407-11e6-9764-5693df39efba.png)


# Propuesta_4
![logo_chapter_app_v4-01](https://cloud.githubusercontent.com/assets/19508456/22356351/0741dc52-e407-11e6-9cc6-57a5f6c240b9.png)


# Propuesta_5
![logo_chapter_app_v5-01](https://cloud.githubusercontent.com/assets/19508456/22356352/0764b092-e407-11e6-80d6-f1be840747f2.png)


# Propuesta_6
![logo_chapter_app_v6-01](https://cloud.githubusercontent.com/assets/19508456/22356756/1bab89ce-e40a-11e6-8f81-90b4501327ae.png)


@jhonmarquez92 @ivonnesantos

**Comment by proseuz** [comment]
Yo me inclino por este, pero mejoraría los espacios, sería bueno separar un poco la imagen del logotipo y un poco también la palabra APP, incluso puede estar más grande, reduce el kerning por favor
![image](https://cloud.githubusercontent.com/assets/17308060/22358364/dac11964-e414-11e6-9a81-2ce31375ae05.png)

**Comment by proseuz** [comment]
deberíamos usar estos colores que son los de las licencias mineras @ivonnesantos 
![image](https://cloud.githubusercontent.com/assets/17308060/22358463/a627d43a-e415-11e6-96ae-c20125fe5066.png)

**Comment by rosanacoronado** [comment]
@jhonmarquez92 diferentes variaciones con el logo pre-seleccionado, reduje el kerning, acomode algunas astas de la palabra chapter de manera que tuvieran similitud con las esquinas curvas de la imagen. Y planteo juego de colores u otras maneras mas simples de trabajar con esa misma idea

Propuesta 5.1
![logo_chapter_app_v5_1-09](https://cloud.githubusercontent.com/assets/19508456/22364009/cc709922-e444-11e6-8242-adbc27279652.png)

Propuesta 5,2
![logo_chapter_app_v5_2-1-08](https://cloud.githubusercontent.com/assets/19508456/22364011/cc812e18-e444-11e6-9a5d-2f65d289021f.png)

Propuesta 5.3
![logo_chapter_app_v5_2-2-08](https://cloud.githubusercontent.com/assets/19508456/22364012/cc99f934-e444-11e6-81cb-7276e3ab798b.png)

Propuesta 5.4
![logo_chapter_app_v5_2-08](https://cloud.githubusercontent.com/assets/19508456/22364013/ccaafb30-e444-11e6-8916-8f382c9e34f7.png)

Propuesta 5.5
![logo_chapter_app_v5_3-07](https://cloud.githubusercontent.com/assets/19508456/22364014/ccc48fe6-e444-11e6-9edb-6219f4a21dd6.png)

Propuesta 5.6
![logo_chapter_app_v5_4-06](https://cloud.githubusercontent.com/assets/19508456/22364015/ccd39900-e444-11e6-8b21-0029b653d7dc.png)

Propuesta 5.7
![logo_chapter_app_v5_5-05](https://cloud.githubusercontent.com/assets/19508456/22364016/cce70ec2-e444-11e6-8f72-923c61acb24c.png)

**Comment by proseuz** [comment]
prueba esta con la sombra completa 
![image](https://cloud.githubusercontent.com/assets/17308060/22390580/3c8d5a4c-e4c2-11e6-90ae-ee52011b9f77.png)
prueba esta con el azul
![image](https://cloud.githubusercontent.com/assets/17308060/22390588/565e5476-e4c2-11e6-8439-f060a54fe431.png)

**Comment by proseuz** [comment]
@ivonnesantos

**Comment by rosanacoronado** [comment]
@jhonmarquez92  @ivonnesantos cambios sugeridos

Propuesta_6
![logo_chapter_app_v5-1-09](https://cloud.githubusercontent.com/assets/19508456/22391233/89c3509c-e4c6-11e6-8834-a4510b728bfe.png)

Propuesta_7
![logo_chapter_app_v5-4-10](https://cloud.githubusercontent.com/assets/19508456/22391580/6cd704a4-e4c8-11e6-8d0d-1c51794ad41f.png)

**Comment by proseuz** [comment]
yo me quedo con la azul. aunque le pondría la sombra un tono más claro. cuando Ivonne se conecte esperamos la opinión de ella @ivonnesantos @rosanacoronado @emmysm  Emmy también me gustaría su opinión

**Comment by rosanacoronado** [comment]
@jhonmarquez92 ajuste de color en la sombra

![logo_chapter_app_v7_](https://cloud.githubusercontent.com/assets/19508456/22395384/49d4d80a-e513-11e6-85ae-acf0f5fbd354.png)

## Iconos
Version_1
![icon_chapter_app_72x72_4-01](https://cloud.githubusercontent.com/assets/19508456/22395389/6f9d16d8-e513-11e6-8ce3-6177c64f2b1e.png)

![icon_mockup_chapter_app_v1](https://cloud.githubusercontent.com/assets/19508456/22395412/288288ea-e514-11e6-87d7-4b05b22594b3.jpg)


Version_2
![icon_chapter_app_72x72-02](https://cloud.githubusercontent.com/assets/19508456/22395422/4f81aeb2-e514-11e6-9eca-8b60d3f0e72d.png)

![icon_mockup_chapter_app_v2](https://cloud.githubusercontent.com/assets/19508456/22395423/597108fa-e514-11e6-8adb-3a9707a394a2.jpg)

**Comment by ivonnesantos** [comment]
@rosanacoronado  en este caso me gusta el de colores la propuesta 6 porque se relaciona con las licencias y con el logo de fermat que tambien tiene colores de alta saturacion @jhonmarquez92 ya le puse a luis el enlace para ver que opina el pero yo en definitiva me inclino por la 6 sin el circulo

**Comment by emmysm** [comment]
Estoy de acuerdo con Ivonne. Me gustaría ver la propuesta 7 con los colores de la 6

**Comment by rosanacoronado** [comment]
Cambios sugerido por Emmy en las propuesta 6
![logo_chapter_app_v7_2_mesa de trabajo 15 copia](https://cloud.githubusercontent.com/assets/19508456/22448284/c76eff3a-e72e-11e6-8fa9-1bf26c601aa5.png)

![chapter splash screen 5-06](https://cloud.githubusercontent.com/assets/19508456/22448612/f0ce554a-e730-11e6-83b2-7f5d29363137.png)

Propuesta_8
![logo_chapter_app_v8_mesa de trabajo 15 copia 6](https://cloud.githubusercontent.com/assets/19508456/22448285/c79a3678-e72e-11e6-9d14-ec37457b9844.png)
 
![chapter splash screen 3-06](https://cloud.githubusercontent.com/assets/19508456/22448658/592cc7ca-e731-11e6-8d00-c659ca0607a9.png)


Propuesta_9
![logo_chapter_app_v9_mesa de trabajo 15 copia 7](https://cloud.githubusercontent.com/assets/19508456/22448286/c7c57a40-e72e-11e6-8346-fa3408333ce8.png)

![chapter splash screen 4-06](https://cloud.githubusercontent.com/assets/19508456/22448668/619b7f82-e731-11e6-8497-54dc8d47ee39.png)

@jhonmarquez92 @emmysm @ivonnesantos

**Comment by ivonnesantos** [comment]
@rosanacoronado coloca la 6 original junto a las 6 que sugiere emmy en el telefono

**Comment by rosanacoronado** [comment]
@ivonnesantos @jhonmarquez92 @emmysm acá esta la pantalla donde se puede apreciar ambas propuestas (lado derecho versión original)

![chapter splash screen 6](https://cloud.githubusercontent.com/assets/19508456/22491195/3addda62-e7f8-11e6-94a9-efe0529a32e9.png)

**Comment by proseuz** [comment]
@rosanacoronado   el de la derecha
